Stock-signalling.py

```diff
@@ -0,0 +1,57 @@
+import requests
+
+
+# List ofstock symbols that will be monitored
+stock_symbols = ["TSLA", "AAPL", "MSFT", "GOOGL", "NKE"]
+
+# API key and the base URL
+alpha_vantage_api_key = "NZXLRSMAPCX4T3YW"
+base_url = "https://www.alphavantage.co/query"
+
+# IFTTT Webhook URL
+ifttt_webhook_url = "https://maker.ifttt.com/trigger/stock-price-decrease/json/with/key/bWs2poP5Q2RliJ9-PmAYcbb"
+
+# Dictionary will store the previous stock prices
+previous_prices = {}
+previous_price = 0 
+
+def get_stock_price(symbol):
+   # Get the daily closing stock price for a given symbol using the Alpha Vantage API.
+    params = {
+        "function": "TIME_SERIES_DAILY",
+        "symbol": symbol,
+        "apikey": alpha_vantage_api_key
+        }
+    for symbol in stock_symbols:
+        stock_price = get_stock_price(symbol)
+        previous_price = previous_prices.get(symbol, None)

```

Comment:
I like that you added none, in case there was no previous price available, it shows that you are thinking of the user experience.